oozie-to-airflow/examples/spark/workflow.xml

```diff
@@ -39,14 +39,14 @@
             <resource-manager>${resourceManager}</resource-manager>
             <name-node>${nameNode}</name-node>
             <prepare>
-                <delete path="${nameNode}/user/${user.name}/${examplesRoot}/output-data/spark"/>
+                <delete path="${nameNode}/user/spark/examples/spark/oozie-examples-4.3.0-copy.jar"/>
```

Comment:
Why do we have spark examples here? the example should be self-sufficient and should not require the .jar file in specific directory. Can we add a file to copy to "assets" folder and copy it from there ? This is much better because we can execute the test automatically

Reply:
This is a result artifact. I have to specify where I will store files.  This line provides idempotence. I will look at how to now ensure the availability of this library using your solution.

Reply:
I updated a PR. Thanks.